### PR TITLE
chore: add funcitonality to resolve variables values

### PR DESCRIPTION
**Pull request by joneff**
* to be used for the documentation
* since sass-variables-parser is unmaintained, there is some trickery applied to bootstrap files

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -114,3 +117,42 @@ gulp.task("docs:check", function() {
     });
 });
 // #endregion
+
+
+gulp.task("resolve-vars", function() {
+    let themes = glob.sync(paths.sass.themes);
+    const { parse } = require('sass-variable-parser');
```
Comment (PreslavKozovski) [inline gulpfile.js:124]: I believe we should hoist this var.

File: gulpfile.js [hunk 2]
```diff
@@ -114,3 +117,42 @@ gulp.task("docs:check", function() {
...
+    const cwd = process.cwd();
+
+    themes.forEach( theme => {
+
```
Comment (PreslavKozovski) [inline gulpfile.js:128]: What JS guidelines do we use for this repo? In Kendo for jQuery, we do not add an empty line at the beginning and the end of a function, is this okay for this repo?

File: gulpfile.js [hunk 3]
```diff
@@ -114,3 +117,42 @@ gulp.task("docs:check", function() {
...
+        let content = fs.readFileSync( variablesScss, 'utf-8' );
+
+        content = content.replace(/ url\("data.*?\)/g, 'none');
+        content = content.replace(/\/\/.*/gm, '');
+        content = content.replace(/\n/gm, '');
+        content = content.replace(/;/gm, ';\n');
+
+        const variables = parse( content, { camelCase: false } );
+
+        fse.ensureFileSync( variablesJson );
```
Comment (PreslavKozovski) [inline gulpfile.js:152]: I believe this does nothing, and we can remove it.

File: gulpfile.js (hunk 2 above)
Comment (joneff) [reply gulpfile.js:128]: we'll have to think on that.

**Pull request merged**